.github/workflows/codemod_publish.yml

```
@@ -103,14 +107,11 @@ jobs:
         working-directory: ${{ steps.parse-tag.outputs.codemod-path }}
         run: node --test
 
-      - name: Authenticate with Codemod registry
-        env:
-          CODEMOD_TOKEN: ${{ secrets.CODEMOD_TOKEN }}
-        run: npx codemod login --api-key "$CODEMOD_TOKEN"
 
-      - name: Publish codemod
-        working-directory: ${{ steps.parse-tag.outputs.codemod-path }}
-        run: npx codemod publish
+      - name: Publish codemod to registry
+        uses: codemod/publish-action@dd6c8dbc5ceb1a6146feba41481d88b43da50024
```

Add the version at the end

```suggestion
        uses: codemod/publish-action@dd6c8dbc5ceb1a6146feba41481d88b43da50024 # v1
```